Automattic/wp-calypso
# Sensei Tailored Flow

#### Proposed Changes

* Adds a Sensei Tailored flow.

p6rkRX-4VM-p2

#### Testing Instructions



* Go to `calypso.localhost:3000/setup/sensei` to kickoff the Sensei Tailored flow.
* Confirm everything works as expected.

You can also test this out with the calypso live link here: https://container-ecstatic-khayyam.calypso.live/setup/sensei

#### Pre-merge Checklist
